src/silicone/utils.py

```diff
@@ -250,3 +257,185 @@ def _get_unit_of_variable(df, variable, multiple_units="raise"):
         return units
 
     return units
+
+
+def return_cases_which_consistently_split(
+    df, to_split, components, how_close=None, use_AR4_data=False
+):
+    """
+    Returns model-scenario tuples which correctly split up the to_split into the various
+    components. Components may contain wildcard "*"s to match several variables.
+    Parameters
+    ----------
+    df: :obj:`pyam.IamDataFrame`
+        The input dataframe.
+
+    to_split : str
```

Comment:
if the other argument is called 'components', perhaps this could be called 'aggregate' (as in 'basket')

Reply:
ok, changed